keyboards/nomu30/nomu30.h

```diff
@@ -16,33 +16,8 @@
 #pragma once
 
 #include "quantum.h"
-
-/* This a shortcut to help you visually see your layout.
- *
- * The first section contains all of the arguments representing the physical
- * layout of the board and position of the keys.
- *
- * The second converts the arguments into a two-dimensional array which
- * represents the switch matrix.
- */
-#define LAYOUT( \
-        K001, K002, K003, K004, K005, K006, K007, K008, K009, K010, K011, \
-  K100, K101, K102, K103, K104, K105, K106, K107, K108, K109, K110,       \
-  K200, K201, K202, K203, K204, K205, K206, K207, K208                    \
-) \
-{ \
-  { KC_NO, K001, K002, K003, K004, K005, K006, K007, K008, K009,  K010,  K011  }, \
-  { K100,  K101, K102, K103, K104, K105, K106, K107, K108, K109,  K110,  KC_NO }, \
-  { K200,  K201, K202, K203, K204, K205, K206, K207, K208, KC_NO, KC_NO, KC_NO }, \
-}
```

Comment:
Correct me if I'm wrong, but the LAYOUT macro seems to be the same for both revisions.
In this case, I'd leave it here to reduce code duplication.

Reply:
Thanks, fixed. 1840c15b0